### DISCUSSION TITLE
希望开发大佬能做一个智能/自定义弹幕屏蔽等级的功能

### DISCUSSION THREAD

**Discussion by 4AMNyaruko** [the1812/Bilibili-Evolved#3445] [功能建议 / Ideas]
例如：视频已有弹幕数<200时，屏蔽等级=0；200<弹幕数<1000时，屏蔽等级=5；弹幕数>1000时屏蔽等级=10，或硬核屏蔽（若有）。当然具体屏蔽划分数可以进一步优化，或由用户自定义。谢谢！

**Comment by Geeyun-JY3** [comment]
其实是不是可以到隔壁问问：https://github.com/xmcp/pakku.js
https://github.com/xmcp/pakku-advanced-filter

**Reply by the1812** [reply]
有道理(
